model/src/w3gridmd.F90

```diff
@@ -2432,8 +2438,11 @@ SUBROUTINE W3GRID()
     JGS_TERMINATE_NORM = .FALSE.
     JGS_LIMITER = .FALSE.
     JGS_LIMITER_FUNC = 1
+    JGS_GSE_TS = 350000
     JGS_BLOCK_GAUSS_SEIDEL = .TRUE.
     JGS_USE_JACOBI = .TRUE.
+    JGS_LGSE = .FALSE. 
```

Comment:
Please add descriptions of the new terms to model/inp/ww3_grid.inp in the UNST namelist section: https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_grid.inp#L318

Reply:
Thanks for adding updates to ww3_grid.inp !